testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -46,6 +41,13 @@ Feature: Disable and re-enable monitoring of the server
     And I should see a list item with text "Taskomatic (Java JMX)" and a failing bullet
     And I should see a list item with text "Tomcat (Java JMX)" and a failing bullet
     And I should not see a "Restarting Tomcat and Taskomatic is needed for the configuration changes to take effect." text
+    And file "/etc/rhn/rhn.conf" should contain "prometheus_monitoring_enabled = 0" on server
+    And file "/usr/lib/systemd/system/tomcat.service.d/jmx.conf" should not exist on server
+    And file "/usr/lib/systemd/system/taskomatic.service.d/jmx.conf" should not exist on server
```

Comment:
Nono, what I said is that it make sense to me to move them here if we already have a scenario specifically checking the outcome of the previous scenario.

BUT, I clarified that the BDD best practices tell as to have a unique scenario that will have : Steps to check preconditions, A unique business action (it could be fill a form and click submit), steps to check the outcome from this action.

But for me, for the way we make it here, ok, make sense to just move these checks too.